external/build/lib/dts-parse.js

```diff
@@ -327,6 +327,9 @@ class Transform {
     if (!this.filter(node, parent, namespace)) {
       return null;
     }
+    for (const source of node.sources ?? []) {
+      source.fileName = ''; // HACK
```

Comment:
Please add a cause for this hack, ie that we want to avoid duplicate builds

Reply:
This was actually my 1st attempt that was superceded by the code below. I've removed this and updated that comment even further.